### PR TITLE
chore(various): remove beta status

### PR DESCRIPTION
**Pull request by vdice**
# Summary of Changes

Docs change in `README.md` and defaults change in `pkg/gitreceive/config.go`.

Ref deis/workflow#285

# Pull Request Hygiene TODOs

Please make sure the below checklist is complete.

- [x] Your pull request is concise and to the point (make another PR for refactoring nearby code)
- [x] Your commits follow the [commit style guidelines](http://docs.deis.io/en/latest/contributing/standards/#commit-style-guide)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
 # Deis Builder v2
 
-[![Build Status](https://travis-ci.org/deis/builder.svg?branch=master)](https://travis-ci.org/deis/builder) [![Go Report Card](http://goreportcard.com/badge/deis/builder)](http://goreportcard.com/report/deis/builder) [![codebeat badge](https://codebeat.co/badges/e29e5e2b-531d-4374-810b-f05053c47688)](https://codebeat.co/projects/github-com-deis-builder) [![Docker Repository on Quay](https://quay.io/repository/deisci/builder/status "Docker Repository on Quay")](https://quay.io/repository/deisci/builder) 
+[![Build Status](https://travis-ci.org/deis/builder.svg?branch=master)](https://travis-ci.org/deis/builder) [![Go Report Card](http://goreportcard.com/badge/deis/builder)](http://goreportcard.com/report/deis/builder) [![codebeat badge](https://codebeat.co/badges/e29e5e2b-531d-4374-810b-f05053c47688)](https://codebeat.co/projects/github-com-deis-builder) [![Docker Repository on Quay](https://quay.io/repository/deisci/builder/status "Docker Repository on Quay")](https://quay.io/repository/deisci/builder)
```
Comment (arschles) [inline README.md]: as long as you're removing a space here, can you split these up onto separate lines?

**Comment by arschles** [comment]
code LGTM, but this shouldn't be merged until v2 stable is released, right?

File: README.md (hunk 1 above)
Comment (vdice) [inline README.md]: @arschles done in latest commit

**Comment by bacongobbler** [comment]
yes, especially since this relies on a v2.0.0 image for `SlugbuilderImage` and `DockerbuilderImage`.

**Comment by vdice** [comment]
I've narrowed the scope of this change do only the `README.md` update and created a separate issue tracking the updates to `pkg/gitreceive/config.go` in https://github.com/deis/builder/issues/357

**Pull request merged**